flow_screen_components/lookupFSC/README.md

```diff
@@ -16,23 +16,63 @@ See the full set of installation options [here](/install.md).
 
 This component exposes the following attributes that can be set in Flow:
 
-label="Object Name" This is the actual object that will be looked up
+label="I1_Object Name" This is the actual object that will be looked up
```

Comment:
what's the reason for adding the prefixes? they complicate the naming a little. Do they add value?

Reply:
I added the prefixes to identify input vs output parameters and to try and identify a logical sequence for assigning values